tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -38,47 +38,11 @@ page in the Mono documentation.
 Setting up C# for Godot
 -----------------------
 
-Windows (Visual Studio)
-~~~~~~~~~~~~~~~~~~~~~~~
-
-Download and install the latest version of
-`Visual Studio <https://visualstudio.microsoft.com/downloads/>`_
-(*not* Visual Studio Code), which contains utilities required to use
-C# in Godot. If you don't plan on using the Visual Studio IDE,
-you can download just the
-`Visual Studio Build Tools <https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=15>`_
-instead.
-Make sure you at least have the .NET Framework 4.5 targeting pack installed, you can get it using any of the installers mentioned above inside the "Individual components" tab.
-
-Windows (JetBrains Rider)
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-JetBrains Rider comes with bundled MSBuild, so nothing extra is required.
-Make sure to set the following preferences:
-
-- In Godot's Editor Settings:
-
-   - Set **Mono External Editor** to **JetBrains Rider**.
-   - set **Mono Build Tool** to **JetBrains Mono**.
-
-- In Rider:
-
-   - Set **MSBuild version** to either **Bundled with Rider** or **.NET Core**.
-   - Install the **Godot support** plugin.
-
-macOS and Linux
-~~~~~~~~~~~~~~~
+Prerequisites
+~~~~~~~~~~~~~
 
-Download and install the latest version of the
-`Mono SDK <http://www.mono-project.com/download/>`_. As of Godot 3.1 beta 3,
-the version number doesn't matter since Godot bundles its own Mono 5.18
-installation. We only need the Mono installation for NuGet and MSBuild
-which are required to use C# in Godot.
-
-.. note:: To download Mono on macOS, use the "Stable Channel" link
-          from the `Mono Downloads Page <http://www.mono-project.com/download/>`_.
-          The Visual Studio channel is an earlier version of Mono and
-          will not work.
+Install the latest stable version of `.NET Core <https://dotnet.microsoft.com/download/dotnet-core>`__ (3.1 as of writing).
+As of Godot 3.2.3, installing Mono is not a requirement anymore.
```

Comment:
Are you sure Mono is not a requirement anymore? I've encountered many situations where C# in Godot wasn't working, then I tell people to install Mono and then it works.

Why mention .NET Core? Godot doesn't use .NET Core, and I can run Godot just fine without .NET Core. Is it just that you can install either Mono or .NET Core and it will work?

Anyway, if kept, the mention of .NET Core should be updated for .NET 5.0. While the "Core" is no longer part of the branding, we may wish to include it anyway so that people know we don't mean .NET Framework.

Reply:
Suggesting everybody to use net 5 should wait for resolution of godotengine/godot#43717

Reply:
Installed Mono isn't used by Godot anymore. It's only needed for building Godot itself.

>Why mention .NET Core? Godot doesn't use .NET Core, and I can run Godot just fine without .NET Core. Is it just that you can install either Mono or .NET Core and it will work?

The docs should say .NET Core SDK, instead of just .NET Core as users could mistakenly download the runtime-only version. The .NET Core SDK is needed for building C# projects with the newer Sdk style (even when using Visual Studio's MSBuild you need to install the .NET Core Visual Studio workload).

Reply:
`.NET Core` updated to `.NET Core SDK`.
Commits squashed.